assets/large_language_models/autoprompt/environments/autoprompt/context/conda_dependencies.yaml

```diff
@@ -20,9 +20,9 @@ dependencies:
   - scikit-learn==0.24.1
   - mltable
   - transformers==4.21.1
-  - torch==1.11
-  - openai
-  - azure-ai-ml
+  - torch~=1.11
+  - openai~=0.27.4
+  - azure-ai-ml~=1.4.0
```

Comment:
Does azure-ai-ml have to be pinned to ~=1.4.0 ?
That version is from Feb 6, 2023.
The latest version is 1.8.0